RELEASE-NOTES.md

```diff
@@ -1,5 +1,88 @@
+   pass
+   
+@flow(on_crashed=[my_first_hook, my_second_hook])
+def my_flow():
+   # call two different hooks if this flow enters a `CRASHED` state
+   pass
+```
+
+See the [pull request](https://github.com/PrefectHQ/prefect/pull/9418) for implementation details.
+
+### Enhancements
+- Prevent unnecessarily verbose logs by updating `log_prints` to ignore prints where a custom `file` is used — https://github.com/PrefectHQ/prefect/pull/9358
+- Create a process work pool by default when a new worker is started with a new work pool name and no type — https://github.com/PrefectHQ/prefect/pull/9326
+- Add support for asyncronous project steps — https://github.com/PrefectHQ/prefect/pull/9388
+- Update `critical_service_loop` to retry on all 5XX HTTP status codes — https://github.com/PrefectHQ/prefect/pull/9400
+- Add backoff on failure to agent critical loop services — https://github.com/PrefectHQ/prefect/pull/9402
+- Add print statement to `git pull` to isolate issues between clone and execution — https://github.com/PrefectHQ/prefect/pull/9328
+- Add `on_crashed` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9418
+- Make build->push step explicit in docker project recipes — https://github.com/PrefectHQ/prefect/pull/9417
+- Add storage blocks to cli `deployment build` help description  — https://github.com/PrefectHQ/prefect/pull/9411
+- Add `call_in_...` methods to the concurrency API — https://github.com/PrefectHQ/prefect/pull/9415
+- Add support for `Callable[[], T]` to concurrency API methods — https://github.com/PrefectHQ/prefect/pull/9413
+- Add `timeout` to `QueueService.drain_all` and limit `APILogHandler.flush` time to 5s — https://github.com/PrefectHQ/prefect/pull/9354
+- Add a parameters JSON input option for deployments in the UI — [`#1405`](https://github.com/PrefectHQ/prefect-ui-library/pull/1405)
+- Improve consistency in UI help modals — [`#1397`](https://github.com/PrefectHQ/prefect-ui-library/pull/1397)
+
+### Fixes
+- Add guard against null schedule in `deployment.yaml` — https://github.com/PrefectHQ/prefect/pull/9373
+- Fix issue preventing work pool filter from being applied to the flow runs page — https://github.com/PrefectHQ/prefect/pull/9390
+- Fix project recipe `image_name` and `tag` templating in docker-git, docker-gcs, and docker-s3 — https://github.com/PrefectHQ/prefect/pull/9425
+- Fix bug with work queues showing as unhealthy when a work queue with the same name is unhealthy — https://github.com/PrefectHQ/prefect/pull/9437
+- Fix bug where child flows would not fail the parent when they received invalid arguments — https://github.com/PrefectHQ/prefect/pull/9386
+- Fix schema values mapping on the create flow run forms to ensure all parameter values can be edited — [`#1407`](https://github.com/PrefectHQ/prefect-ui-library/pull/1407)
+- Add a check for color scheme to ensure the flow run state favicon is visible — [`#1392`](https://github.com/PrefectHQ/prefect-ui-library/pull/1392)
+
```

Comment:
```suggestion
- Fix send/drain race conditions in queue services — https://github.com/PrefectHQ/prefect/pull/9426
- Fix bug where missing trailing slash in remote filesystems path would cause download failures — https://github.com/PrefectHQ/prefect/pull/9440

```